### DISCUSSION TITLE
General list of names

### DISCUSSION THREAD

**Discussion by BobvanSchendel** [mortii/anki-morphs#217] [Q&A]
Does anyone have a list of names in general or within certain regions to plug into ankimorphs? I'm having trouble finding/generating one myself.

**Comment by Vilhelm-Ian** [comment]
For which language do you need

**Reply by BobvanSchendel** [reply]
Swedish. I just had an idea of effectively scraping a big database so if I do that I don't mind sharing it here. Do you know if there is a place in this repo or somewhere else where I could share that?

**Reply by Vilhelm-Ian** [reply]
that depends on morti. He already has uploaded frequency lists. So he might be open to names.

I was asking because I thought I could easily get a list like that for you but it seems you already figured it out

**Reply by mortii** [reply]
Share them here if you want and I can place them in the guide in the same way as the frequency files :+1:

**Comment by mortii** [comment]
I added a comment in the [names.txt guide](https://mortii.github.io/anki-morphs/user_guide/setup/names.html#names) about how large lists of names are inefficient, so this I think it would probably best to just add names on the fly as it becomes necessary.
